### PR TITLE
Features/#15 osmtgmod to pypsa

### PR DESCRIPTION
**Pull request by ulfmueller**
integrates the old data processing step "ego_dp_powerflow_osmtgmod_to_pypsa.sql" into the work flow.

The version and the scenario name implementation is very beta and should be tackled at some point.

### CODE REVIEW

File: src/egon/data/processing/osmtgmod/osmtgmod_to_pypsa.sql [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+	(SELECT bus0 FROM grid.egon_pf_hv_line WHERE scn_name='Status Quo')
+AND bus_id NOT IN 
+	(SELECT bus1 FROM grid.egon_pf_hv_line WHERE scn_name='Status Quo')
+AND bus_id NOT IN 
+	(SELECT bus0 FROM grid.egon_pf_hv_transformer WHERE scn_name='Status Quo')
+AND bus_id NOT IN 
+	(SELECT bus1 FROM grid.egon_pf_hv_transformer WHERE scn_name='Status Quo'); 
+	
+
+/*
```
Comment (IlkaCu) [inline src/egon/data/processing/osmtgmod/osmtgmod_to_pypsa.sql:108]: Is there any need or benefit to keep the code which is commented out?

Comment (ulfmueller) [reply src/egon/data/processing/osmtgmod/osmtgmod_to_pypsa.sql]: u r right. I skipped that part in the newest version.

**Comment by ClaraBuettner** [comment]
I tested this branch and everything worked fine. The result looks good (the topology is a little different compared to our former grid model, but that's another task...)
Could you add a note to the changelog?

**Comment by ulfmueller** [comment]
I think the task is to minor for a changelog entry, I would say....

**Pull request merged**